src/extendPrototype.ts

```diff
@@ -1,13 +1,39 @@
 /* eslint-disable @typescript-eslint/ban-types */
-export default function extendProtoype(
-    target: Function,
-    func: Function,
-    functionName: string
+export function extendProtoype(
+  target: Function,
+  func: Function,
+  functionName: string
 ): void {
-    if (!Object.getOwnPropertyNames(target.prototype).includes(functionName)) {
-        Object.defineProperty(target.prototype, functionName, {
-            value: func,
-            writable: true,
-        })
-    }
+  if (!Object.getOwnPropertyNames(target.prototype).includes(functionName)) {
+    Object.defineProperty(target.prototype, functionName, {
+      value: func,
+      writable: true,
```

Comment:
Why `true` instead of `false`? I thought we want the runtime to explode of others try conflicting stuff, so we do not break their code?